__tests__/unit.test.js

```diff
@@ -0,0 +1,38 @@
+const db = require("../db/connection");
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+afterAll(() => {
+    db.end();
+  });
+  
+  beforeEach(() => {
+    return seed(data);
+  });
+
+describe("GET /api/topics", () => {
+  test('GET: 200 - responds with an array of topic objects with "slug" and "description" properties', () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.topic)).toBe(true);
+        body.topic.forEach((topic) => {
```

Comment:
As we're using a forEach here, could you pop an expect beforehand to check the length please?